### PR TITLE
Issue 19290 select all files broken in content search

### LINKED ISSUE #19290: Card view in content search loading in the background
**Describe the bug**
When you load the content search portlet in list view you can see some images broken in the console, those images are from the card view which should not be loaded if the user don't ask for it.

**To Reproduce**
1. Go to demo.dotcms.com content search portlet (in list view mode)
2. Open the browser console
3. See error

Screen Shot 2020-09-21 at 12 47 25

**Expected behavior**
Card view should only load when users ask for it or when is set in the local storage

## Acceptance Criteria

1. Make sure card view load on demand.
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
Select all files is broken in Content Search if you try to use it before loading the card view.
https://user-images.githubusercontent.com/934364/129929355-07eae87a-a768-4ffc-9189-f035c980c847.png

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/maintenance/JVMInfoResource.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 @SuppressWarnings("serial")
 public class JVMInfoResource implements Serializable {
 
-    final Pattern obfuscatePattern = Pattern.compile(Config.getStringProperty("OBFUSCATE_SYSTEM_ENVIRONMENTAL_VARIABLES", "/passw|pass|secret|key|pwd/gi"),Pattern.CASE_INSENSITIVE);
+    final Pattern obfuscatePattern = Pattern.compile(Config.getStringProperty("OBFUSCATE_SYSTEM_ENVIRONMENTAL_VARIABLES", "passw|pass|passwd|secret|key"),Pattern.CASE_INSENSITIVE);
```
Comment (nollymar) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/maintenance/JVMInfoResource.java:37]: why this change? It seems that your branch is out of sync and this PR should be against release-21.09

**Review by nollymar — CHANGES_REQUESTED** [review]

**Review by jdotcms — APPROVED** [review]

**Review by nollymar — APPROVED** [review]
